### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by Chrissiku**
## What I did ♻️ 

> In this project's branch I implement the API consumption with redux toolkit and display the list of all rooms to the users.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,31 +1,36 @@
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import { SignedInRoute } from './Components/middlewares';
 import {
-  Signin, Register, PasswordReset, NewPassword,
+  Signin,
+  Register,
+  PasswordReset,
+  NewPassword,
 } from './Components/authentication';
```
Comment (sntakirutimana72) [inline src/App.js:8]: These components are inside `src/components/` not `src/Components/`. Kindly fix that as it's causing conflicts with your linters.

**Review by sntakirutimana72 — CHANGES_REQUESTED** [review]
Hi @Chrissiku,

# Status: CHANGES REQUIRED

You've done an amazing job so far, but there are some few flaws you have to improve.

## Highlight! :green_circle: 

:heavy_check_mark:  Responsive design
:heavy_check_mark:  Good topography
:heavy_check_mark:  Good commit messages

## [REQUIRED] changes

- While rendering data from the `API`, you seem to duplicate it and therefore end up creating duplicate `UI` features. Kindly, fix this.
- Kindly, fix linter errors. Linters help you write clean code and apply best coding practices.

## [Optional] suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

-----

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack using this handle @sntakirutimana72.

**Review by sntakirutimana72 — APPROVED** [review]
Hi @Chrissiku,

# Status: APPROVED!!!  :champagne: 🥂

Bravo👏!!! You've done a great job implementing all required behaviors.👍

## Highlight! :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  No linter errors.

## [Optional] suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

-----

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack using this handle @sntakirutimana72.

**Review by wale-prog — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Chrissiku 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by safar1212 — APPROVED** [review]
# APPROVED!!!! 🎉
## Hey @Chrissiku    😊 😃 

#### Great Job 😄 👏 You've done it. Nice completing this milestone 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aliatayee — APPROVED** [review]
Hi @Chrissiku,

# Status: APPROVED!!! 

You've done a great job implementing all required behaviors.👍 Time to merge!

## Highlight! :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  No linter errors.

**Pull request merged**